src/elements/content-sidebar/activity-feed/version/Version.js

```diff
@@ -40,7 +40,7 @@ const Version = (props: Props): React.Node => {
     const action = selectors.getVersionAction(props);
     const { id, intl, onInfo, version_number, version_promoted } = props;
     // $FlowFixMe
-    const { name } = selectors.getVersionUser(props);
+    const { name } = selectors.getVersionUser(props, intl);
```

Comment:
I believe `name` is empty for PLACEHOLDER_USER. Can we do something like the following?

```
const { name } = selectors.getVersionUser(props);
const uploaderName = uploader_display_name === FILE_REQUEST ? intl.formatMessage(messages.displayName) : uploader_display_name;
const displayName = name || uploaderName || '';
```